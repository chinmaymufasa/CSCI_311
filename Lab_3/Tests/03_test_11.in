2
84
5
2
73
5
2
143
5
2
7
5
2
138
5
-1
