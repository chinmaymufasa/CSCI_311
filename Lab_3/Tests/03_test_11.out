84 
73 84 
73 84 143 
7 73 84 143 
7 73 84 138 143 
